### ISSUE TITLE
Include Pearson symbol in `spglib.get_symmetry_dataset` output

### ISSUE THREAD

**Issue by CompRhys** [spglib/spglib#125]
I am attempting to use spglib to replicate the aflow prototype labels, part of the prototype label they use is the Peason symbol (https://en.wikipedia.org/wiki/Pearson_symbol). It seems relatively easy to construct from items available in the symmetry_dataset but it could be nice to have it included directly as a convenience item.

**Comment by atztogo** [comment]
Can it be defined unambiguously?

**Comment by CompRhys** [comment]
I think so, the two letters depend on the bravais lattice and so can be looked-up from a data table using the spacegroup number. The number is the number of sites in the conventional unit cell which I believe is already returned in the symmetry_dataset.

**Comment by atztogo** [comment]
Do you know precisely how S,A,B,C can be chosen and how rhombohedral is treated precisely?

**Comment by CompRhys** [comment]
On the choice of ABC/S for base centred orthorhombic materials aflow's labels use C for all orientations - others use the S label to indicate that it's base centred without giving an orientation. I would suggest using C for all given my use case but you may want to use S.

For the rhombohedral systems it's less clear how to standardise the definition. I will investigate further and report back

**Comment by atztogo** [comment]
Once spglib supports an option, it can not be updated until next major update. So I would like to support that is very sure or super useful. As far as I look around internet and literatures briefly, I could not figure out a well accepted definition, especially about R. So I want to let users define their own Pearson symbols by themselves.

About the base centrings, some space group types are represented by A not by C as the first choices of international symbols in spglib. So if we choose only C for possible Pearson symbols, it looks not consistent.

In summary, I think it is better not to support Pearson symbols unless someone can propose a well accepted Pearson symbols and can keep it maintained.
